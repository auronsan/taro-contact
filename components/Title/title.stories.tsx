import React from 'react';
import { Title } from './index';

export default {
  title: 'Components/Title',
  component: Title,
};

export const Default = () => <Title>Default Title</Title>;

export const WithCustomClassName = () => (
  <Title className="custom-class">Title with Custom Class Name</Title>
);
