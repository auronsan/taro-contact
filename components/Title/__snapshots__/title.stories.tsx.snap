// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Title Default smoke-test 1`] = `
<main class="roboto-normal">
  <div style="display: flex; justify-content: center; align-items: center; height: 100vh; flex-direction: column;">
    <h2 class="title_title__RhqmI">
      Default Title
    </h2>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`Components/Title WithCustomClassName smoke-test 1`] = `
<main class="roboto-normal">
  <div style="display: flex; justify-content: center; align-items: center; height: 100vh; flex-direction: column;">
    <h2 class="title_title__RhqmI custom-class">
      Title with Custom Class Name
    </h2>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
